HW_3/templates/login.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Login</title>
+</head>
+<body>
+<h1>Login</h1>
+<form method="POST" action="{{ url_for('login') }}">
+    {{ form.csrf_token }}
+    <p>
+        {{ form.name.label }}<br>
+        {{ form.name(size=50) }}
+    </p>
+    <p>
+        {{ form.surname.lable }}<br>
+        {{ form.surname(size=50) }}
+    </p>
+    <p>
+        {{ form.email.lable }}<br>
+        {{ form.email(size=50) }}
+    </p>
+    <p>
+        {{ form.password.lable }}<br>
+        {{ form.password(size=50) }}
```

Comment:
Не отображаются лэйблы
![image](https://github.com/Fejanin/HW_Flask_and_FastAPI/assets/49741194/0111035f-7901-4d14-b35c-3f71ae46829a)

Reply:
Все потому, что правильно писать так
```suggestion
        {{ form.surname.label}}<br>
        {{ form.surname(size=50) }}
    </p>
    <p>
        {{ form.email.label}}<br>
        {{ form.email(size=50) }}
    </p>
    <p>
        {{ form.password.label}}<br>
        {{ form.password(size=50) }}
```